internal/telemetry/cluster_test.go

```diff
@@ -40,6 +40,32 @@ func TestNodeCountInAClusterWithOneNode(t *testing.T) {
 	}
 }
 
+func TestClusterIDRetrievesK8sClusterUID(t *testing.T) {
+	t.Parallel()
+
+	c := newTestCollectorForCluserWithNodes(t, node1, kubeNS)
```

Comment:
nit: typo `newTestCollectorForCluserWithNodes` should be `newTestCollectorForClusterWithNodes`